packages/salesforcedx-vscode-apex/package.json

```diff
@@ -49,9 +52,89 @@
     "test": "node ./node_modules/vscode/bin/test",
     "test:unit": "node ./node_modules/vscode/bin/test"
   },
-  "activationEvents": ["workspaceContains:sfdx-project.json"],
+  "activationEvents": [
+    "workspaceContains:sfdx-project.json",
+    "onCommand:sfdx.force.test.view.showError",
+    "onView:sfdx.force.test.view"
+  ],
   "main": "./out/src",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "test",
+          "title": "Test Viewer",
+          "icon": "media/dep.svg"
+        }
+      ]
+    },
+    "views": {
+      "test": [
+        {
+          "id": "sfdx.force.test.view",
+          "name": "Apex Tests"
+        }
+      ]
+    },
+    "menus": {
+      "view/title": [
+        {
+          "command": "sfdx.force.test.view.run",
+          "when": "view == sfdx.force.test.view",
+          "group": "navigation"
+        },
+        {
+          "command": "sfdx.force.test.view.refresh",
+          "when": "view == sfdx.force.test.view",
+          "group": "navigation"
+        }
+      ],
+      "view/item/context": [
+        {
+          "command": "sfdx.force.test.view.showError",
+          "when": "view == sfdx.force.test.view"
+        },
+        {
+          "command": "sfdx.force.test.view.runSingleTest",
+          "when": "view == sfdx.force.test.view",
+          "group": "inline"
+        }
+      ]
+    },
+    "commands": [
+      {
+        "command": "sfdx.force.test.view.run",
+        "title": "Run Tests",
+        "icon": {
+          "light": "resources/light/play-button.svg",
```

Comment:
Where did we get the resources from? Are they OK from a copyright/IP perspective?

Reply:
We got them mocha sidebar. @ntotten said that there is no problem with using them

Reply:
Could you send the link to that resource so I can look at the license and copyright?
This might seem pedantic but we need to be very clear on this to avoid legal issues.

Reply:
https://github.com/maty21/mocha-sidebar